recipe/emailverification/utils.go

```diff
@@ -6,25 +6,34 @@ import (
 )
 
 func ValidateAndNormaliseUserInput(appInfo supertokens.NormalisedAppinfo, config schema.TypeInput) schema.TypeNormalisedInput {
-	getEmailVerificationURL := *config.GetEmailVerificationURL
+	var typeNormalisedInput schema.TypeNormalisedInput
+	typeNormalisedInput.GetEmailVerificationURL = *config.GetEmailVerificationURL
```

Comment:
what if `config.GetEmailVerificationURL` is `nil`? Then doing *config.GetEmailVerificationURL will lead to a panic?

Reply:
We should make `GetEmailVerificationURL` not a pointer, so the above comment is invalid